src/borg/algorithms/zstd/lib/common/zstd_internal.h

```diff
@@ -261,7 +261,7 @@ typedef enum {
  *         - ZSTD_overlap_src_before_dst: The src and dst may overlap, but they MUST be at least 8 bytes apart.
  *           The src buffer must be before the dst buffer.
  */
-MEM_STATIC FORCE_INLINE_ATTR 
+MEM_STATIC FORCE_INLINE_ATTR
```

Comment:
src/borg/algorithms/zstd/* and some other dirs there is not our code, but 3rd party code we just bundle - we should not modify it for cosmetics.

Reply:
Good catch, i need to exclude them